pom.xml

```diff
@@ -52,6 +56,19 @@
 			<artifactId>guava</artifactId>
 			<version>28.2-android</version>
 		</dependency>
+		<dependency>
+			<groupId>mysql</groupId>
+			<artifactId>mysql-connector-java</artifactId>
+			<version>8.0.30</version>
+			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+			<version>2.7.3</version>
+			<scope>compile</scope>
```

Comment:
same question here

Reply:
I added these dependencies with the IDE, compile it is the default scope. Should I change it to another scope?